resources/views/tiny-editor.blade.php

```diff
@@ -22,6 +22,8 @@ class="relative z-0"
                     tinymce.init({
                         target: $refs.tinymce,
                         language: '{{ $getInterfaceLanguage() }}',
+                        toolbar_sticky: true,
```

Comment:
It's always true. Just change `true` to `'{{ $getToolbarSticky() }}'`.

Reply:
Omg. I'm forgot. fixed.